scripts/build.py

```diff
@@ -566,30 +565,34 @@ def _execute_tasks(tasks, batch_size=24):
             task = remaining_tasks.pop()
             currently_running_tasks.append(task)
             task.start()
-        time.sleep(1)
 
 
-def build_files(source, target, file_hashes):
-    """Minifies all CSS and JS files, removes whitespace from HTML and
-    interpolates paths in HTML to include hashes in source
-    directory and copies it to target.
+def build_files(source, target, file_hashes, file_formats=None):
+    """If no specific file formats is provided, minifies all CSS and JS files,
```

Comment:
Also describe what happens if specific file formats are provided.

Reply:
Added more details for the case of no specific file formats is provided!